test/HealthChecks.Publisher.CloudWatch.Tests/DependencyInjection/RegistrationTests.cs

```diff
@@ -0,0 +1,61 @@
+namespace HealthChecks.Publisher.CloudWatch.Tests.DependencyInjection
+{
+    public class cloud_watch_publisher_registration_should
+    {
+        [Fact]
+        public void add_healthcheck_when_properly_configured_with_default_parameter()
+        {
+            var services = new ServiceCollection();
+            services
+                .AddHealthChecks()
+                .AddCloudWatchPublisher();
+
+            using var serviceProvider = services.BuildServiceProvider();
+            var publisher = serviceProvider.GetService<IHealthCheckPublisher>();
+
+            Assert.NotNull(publisher);
+        }
+
+        [Fact]
+        public void add_healthcheck_when_properly_configured_with_custom_service_check_name()
+        {
+            var services = new ServiceCollection();
+            services
+                .AddHealthChecks()
+                .AddCloudWatchPublisher("serviceCheckName");
+
+            using var serviceProvider = services.BuildServiceProvider();
+            var publisher = serviceProvider.GetService<IHealthCheckPublisher>();
+
+            Assert.NotNull(publisher);
```

Comment:
Maybe check name in this test?

Reply:
The same suggestion for 2 tests below - check some configured options.

Reply:
I am not much familiar with unit tests. I did my best 😄